posts/zzz_DO_NOT_EDIT_hello_admiralmetabolic/hello_admiralmetabolic!.qmd

```diff
@@ -0,0 +1,145 @@
+---
+title: "Hello admiralmetabolic!"
+author:
+  - name: Anders Askeland
+  - name: Edoardo Mancini
+description: "We are happy to announce the release of the newest member of the {admiral} family - {admiralmetabolic}!"
+# Note that the date below will be auto-updated when the post is merged.
+date: "2024-12-01"
+# Please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [ADaM, Community, Technical]
+# Feel free to change the image
+image: "admiralmetabolic.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "zzz_DO_NOT_EDIT_hello_admiralmetabolic"
+library(link)
+link::auto()
+```
+
+<!--------------- post begins here ----------------->
+
+[![](admiralmetabolic.png){fig-align="center" width="220"}](https://pharmaverse.github.io/admiralmetabolic/index.html)
+
+# Introduction
+
+The {admiral} team is excited to celebrate the newest addition to the our extension package family with the release {admiralmetabolic} 0.1! This new package focuses on the therapeutic areas of obesity and cardiovascular metabolism, and was borne out of a collaboration between Novo Nordisk and Roche, as well as a number of independent contributors. {admiralmetabolic} addresses a key unmet need in this area given the ever-increasing interest in obesity and cardiovascular metabolism across the industry.
```

Comment:
```suggestion
The {admiral} team is excited to celebrate the newest addition to our extension package family with the release {admiralmetabolic} 0.1! This new package focuses on the therapeutic areas of obesity and cardiovascular metabolism, and was borne out of a collaboration between Novo Nordisk and Roche, as well as a number of independent contributors. {admiralmetabolic} addresses a key unmet need in this area given the ever-increasing interest in obesity, diabetes, and cardiovascular metabolism across the industry.
```

From my perspective, I would love to implement various diabetes associated functionalities at some point as well.